scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/TaskFilterCriteria.java

```diff
@@ -28,7 +28,7 @@
     private boolean running = true;
     private boolean pending = true;
     private boolean finished = true;
-    private ArrayList<SortParameter<TaskSortParameter>> sortParameters = null;
+    private ArrayList<SortSpecifierContainer> sortParameters = null;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The type of the "sortParameters" object should be an interface such as "List" rather than the implementation "ArrayList". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1319)